NEWS.md

```diff
@@ -1,3 +1,9 @@
+# circacompare 0.1.1.9001
+
+## Improvements
+
+* allow per-sample weights in `circacompare()` and `circa_single()`. This allows to downweight individual samples, for example in the case of outliers rather than hard-filtered them which reduces power. Such per-sample weights can reproducibly be estimated using approaches such as `arrayWeights()` in the [limma](https://bioconductor.org/packages/release/bioc/html/limma.html) package for datasets with many genes/observations, such as RNA-seq or microarrays.
```

Comment:
I'd prefer not to suggest how people handle outliers. In the documentation or in the version history. There are many reasons why people may want to use weights other than handling outliers and I want to be agnostic to these methods. Could you please reword to state the change performed and not the motivation?

Reply:
Removed motivation, only stating the change.